addons/analytic/migrations/10.0.1.1/tests/test_analytic.py

```diff
@@ -0,0 +1,13 @@
+# coding: utf-8
+from openerp.tests.common import TransactionCase
+
+
+class TestAnalytic(TransactionCase)
```

Comment:
Partially fixed on https://github.com/OCA/OpenUpgrade/commit/993e0ecc8028aa20b8cd9e52b774732d15d59ef3. Switching to PR for trying to fix the other error: https://travis-ci.org/OCA/OpenUpgrade/builds/232906089#L428